src/common/Common.UI/SettingsDeepLink.cs

```diff
@@ -91,20 +91,19 @@ public static void OpenSettings(SettingsWindow window, bool mainExecutableIsOnTh
         {
             try
             {
-                var assemblyPath = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
-                var fullPath = new DirectoryInfo(assemblyPath).FullName;
+                var directoryPath = System.AppContext.BaseDirectory;
                 if (mainExecutableIsOnTheParentFolder)
                 {
                     // Need to go into parent folder for PowerToys.exe. Likely a WinUI3 App SDK application.
-                    fullPath = fullPath + "\\..\\PowerToys.exe";
+                    directoryPath = directoryPath + "\\..\\PowerToys.exe";
```

Comment:
Given you change it anyway, can we use Path.Combine?
e.g. Path.Combine(directoryPath, "..", "PowerToys.exe");

I seldomly see this kind of "+" code for handling path or uri

Reply:
Probably one of your another PR has the same situation, I know it is not your intention of fix, but using Path.Combine is more safe way.